presto-product-tests/conf/presto/etc/catalog/hive.properties

```diff
@@ -7,7 +7,7 @@
 
 connector.name=hive-cdh5
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.metastore.thrift.client.socks-proxy=hadoop-master:1080
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
```

Comment:
Here's my PR: https://github.com/prestodb/presto/pull/6283.
Feel free to amend your changes or merge the other one.